scan/lib/scan/options.rb

```diff
@@ -146,6 +146,11 @@ def self.available_options
                                      type: Boolean,
                                      default_value: false,
                                      optional: true),
+        FastlaneCore::ConfigItem.new(key: :suppress_xcode_output,
+                                     env_name: "SUPPRESS_XCODE_OUTPUT",
```

Comment:
This `env_name` should be prefix with "SCAN_" like the rest of the envs 😊 so I think it should be `SCAN_SUPPRESS_XCODE_OUTPUT`

Reply:
Nice catch, thanks!